/RoboChessRos2ParamProvider:
  ros__parameters:
    use_sim_time: false
    
    #supported executor types:
      #SINGLE_THREADED(0)
      #STATIC_SINGLE_THREADED(1)
      #MULTI_THREADED(2)
    ros2_executor_type: 2
    
    #default value of 0 will use number of cpu threads
    #used only for MULTI_THREADED executor
    ros2_executor_threads_num: 2
    
    robot_ip: "192.168.56.101"
    robot_port: 30003
